### ISSUE TITLE
Bottles files missing 

### ISSUE THREAD

**Issue by gaoyifan** [Homebrew/homebrew-versions#1387]
The following bottles files return HTTP 404:

```
http://homebrew.bintray.com/bottles-versions/bazel02-0.2.3.el_capitan.bottle.tar.gz
http://homebrew.bintray.com/bottles-versions/bazel02-0.2.3.yosemite.bottle.tar.gz
http://homebrew.bintray.com/bottles-versions/clang-format38-3.8.0.el_capitan.bottle.tar.gz
http://homebrew.bintray.com/bottles-versions/clang-format38-3.8.0.yosemite.bottle.tar.gz
http://homebrew.bintray.com/bottles-versions/clang-format38-3.8.0.mavericks.bottle.tar.gz
```

Could you please rebottles them?

**Comment by nijikon** [comment]
Fixed in ae36a8f63b0c8ecb3a724d1eaed502a4beef0a5b and 017d165418be63e2a04404ab2770e483878df591

**Comment by gaoyifan** [comment]
@nijikon Thanks a lot😀
